### ISSUE TITLE
Import Function

### ISSUE THREAD

**Issue by acb203** [lordlordy/TrainingDiary#18]
I'm adding a new player - Is it possible to import a list of names to create a squad? If an academy or a squad wants to set this all up, it would be pretty arduous to individually add every player?

**Comment by lordlordy** [comment]
This can be done. Would this list have email addresses ? If not it would be tricky to identify if they're an existing player or not. For instance if you import 'John Smith' and we already have someone by that name how do we know whether it's the existing John Smith or a new one

**Comment by acb203** [comment]
Yes, it would have emails. Our school system has ability to export First, Second name, known as, DOB and email.
